plugins/modules/archive.py

```diff
@@ -71,6 +71,13 @@
       - Remove any added source files and trees after adding to archive.
     type: bool
     default: false
+  reproducible_tar:
+    description:
+      - Set tar metadata and gzip headers to vary less given the same input file content.
+      - Useful for minimizing unneeded archive changes and avoiding handlers that may trigger on such changes.
+    type: bool
```

Comment:
I think as the bare minimum this shouldn't be of type `bool`, but of type `str` with `choices`, so that it's possible to add other ways to make it reproducible later.